qiskit/circuit/__init__.py

```diff
@@ -203,8 +203,8 @@
 
 .. _circuit-module-api:
 
-API overview of :mod:`qiskit.circuit`
-=====================================
```

Comment:
This breaks the docs app header:

<img width="315" alt="Screenshot 2024-05-01 at 5 03 56 PM" src="https://github.com/Qiskit/qiskit/assets/14852634/9f63df62-ac1f-45df-ae0a-0bbf6bbb58d6">